udata/templates/dataset/card.html

```diff
@@ -17,21 +17,20 @@
     <div class="card-body">
         <h4>
             <a href="{{ dataset_url }}" title="{{dataset.title}}">
-                {{ dataset.title|truncate(120) }}
+                {{ dataset.title }}
```

Comment:
I trust you tested with very long titles?

Reply:
Or you're truncating in CSS?

Reply:
They are truncated in CSS indeed :-)